hack/ca/Makefile

```diff
@@ -56,7 +56,7 @@ leaf.crt: int.crt
 		-subj / \
 		-addext "keyUsage = critical, digitalSignature" \
 		-addext "extendedKeyUsage = codeSigning" \
-		-addext "subjectAltName = URI:http://foobar.dev" \
+		-addext "subjectAltName = URI:http://foobar.dev,otherName:1.3.6.1.4.1.57264.1.7;UTF8:FOO" \
```

Comment:
Sets-up a test certificate with two SANs, one with the "URI" type and one with the "otherName" type:

```
SEQUENCE (2 elem)
  OBJECT IDENTIFIER 2.5.29.17 subjectAltName (X.509 extension)
    OCTET STRING (42 byte) 
      SEQUENCE (2 elem)
        [6] (17 byte) http://foobar.dev
        [0] (2 elem)
          OBJECT IDENTIFIER 1.3.6.1.4.1.57264.1.7
          [0] (1 elem)
            UTF8String FOO
```